packages/core-mobile/e2e/configs/reuseStateConfig.json

```diff
@@ -25,17 +25,9 @@
     ]
   },
   "testPathIgnorePatterns": [
-    "./e2e/tests/bridge",
```

Comment:
is this going to run the bridge tests?  i have to ask because that would be bad.

Reply:
the test match should be `<rootDir>/tests/**/*.e2e.ts` and `<rootDir>/tests/**/*.e2e.smoke.ts` and all the files in bridge are ending with `e2e.bridge.ts`
<img width="322" alt="Screenshot 2024-10-02 at 2 43 38 PM" src="https://github.com/user-attachments/assets/f17f2ac0-3b16-466b-9050-362dc13fab7b">